config/known

```diff
@@ -13,6 +13,7 @@
 [ruby-]2.6[.6]
 [ruby-]2.7[.2]
 [ruby-]3[.0.0-preview1]
```

Comment:
It looks like this line should be removed since the next line replaces its function.